app/assets/src/components/PipelineSampleReads.jsx

```diff
@@ -906,8 +861,8 @@ class PipelineSampleReads extends React.Component {
             <div className={cs.pipelineInfo}>
               PIPELINE {version_display} {pipeline_version_blurb}
             </div>
```

Comment:
I also believe this should be part of the header as an optional component. This should all be placeholders that different pages/views can use.